helm/korifi/values.yaml

```diff
@@ -44,6 +44,16 @@ api:
       idle: 900
       readHeader: 10
 
+  infoConfig:
+    build: ""
+    description: ""
+    name: ""
+    version: 0
```

Comment:
Providing the version as a helm value sounds a bit tricky as operators would be able to configure it at their will. What we already do for api and controllers binaries is to make the version available for logging purposes (see [this](https://github.com/cloudfoundry/korifi/blob/c353c9ecd6f55a14799d22bedfaf5a822d15d8cc/version/version.go#L12) variable). Therefore, the /v3/info presenter should return whatever that values is.

Reply:
Not sure what the hisotry behind the property version is, but it seems to me that cf-deployment is setting this property to the major version number of cf-deployment,
see
https://github.com/cloudfoundry/cf-deployment/blob/ab937830c6f4d66d55ae9bc6d2d270255ab53d5c/cf-deployment.yml#L853

Might be a relict here and really better to auto populate it with https://github.com/cloudfoundry/korifi/blob/c353c9ecd6f55a14799d22bedfaf5a822d15d8cc/version/version.go#L12 and change the type to string.

Reply:
As far as I can see the values in cf deployment are already auto populated. I wonder why they needed to expose both `build` and `version` and whether some tools depend on that or if it is just for information purposes as its name implies. If it happens to be the latter I guess we could simply set both `build` and `version` to the same thing as you suggested

Reply:
As far as I can tell, it's just for information purposes, but you never know :-)

I've set `build` and `version` to `version.Version` in the latest commit.